xunit.runner.visualstudio.nuspec

```diff
@@ -41,5 +41,8 @@ Supported platforms:
     <file src="build\xunit.runner.visualstudio.wpa81.targets"                                   target="build\wpa81\xunit.runner.visualstudio.targets" />
     <file src="xunit.runner.visualstudio.wpa81\bin\Release\xunit.runner.visualstudio.wpa81.dll" target="build\wpa81\" />
     <file src="xunit.runner.visualstudio.wpa81\bin\Release\xunit.runner.visualstudio.wpa81.pri" target="build\wpa81\" />
+
+    <file src="xunit.runner.visualstudio.testadapter.dotnetcore\bin\Release\netstandard1.5\xunit.runner.visualstudio.testadapter.deps.json" target="build\netstandard1.5\" />
+    <file src="xunit.runner.visualstudio.testadapter.dotnetcore\bin\Release\netstandard1.5\xunit.runner.visualstudio.testadapter.dll" target="build\netstandard1.5\" />
```

Comment:
> I shouldn't care whether it runs on .NET Core or desktop, as those differences are supposed to be hidden from me.

Except you always run a test on a platform. You can't help it. And PCLs in RockSteady have always run ONLY on the desktop, despite theoretically being able to run everywhere. That's why it's not purely philosophical, and why I'm asking the question about how RockSteady will eventually treat these `netstandard1.5` tests: will it always run them on both Desktop and .NET Core? Will it always choose just one? Will I be forced to choose one or the other?

I think it makes more sense to just say: "Each unit test library is dedicated to a single, specific platform. If you want to use the same test code on many platforms, put it in a shared project which is imported into a bunch of (presumably empty) platform-specific unit test libraries." Nothing could be clearer to your intent than that.